**_Stage 2 task: P53 modelling_**

**_ _**

**Contributors:** Oluchi Onyemeh, Pranjal Paul, Natasha Murape, Evans Boakye , Idoko obinna ****

 

**_Introduction_**

The p53 protein, often referred to as the “guardian of the genome,” plays a crucial role in protecting the integrity of cellular DNA. Encoded by the **TP53 gene**, it is responsible for regulating the cell cycle, DNA repair, apoptosis, and cellular senescence. The p53 protein is a tetrameric sequence-specific DNA-binding transcription factor that plays a central role in the prevention of neoplastic transformation. Oligomerization appears to be essential for the tumor suppressing activity of p53 because oligomerization-deficient p53 mutants cannot suppress the growth of carcinoma cell lines. Mutations in p53 are associated with more than 50% of human cancers, making it one of the most studied proteins in oncology.

P53 is a multi-domain protein consists of several distinct regions:

**N-terminal transactivation domain (TAD)** - Responsible for interacting with transcriptional machinery.

**DNA-binding domain (DBD)**- Recognizes specific DNA sequences, crucial for its transcriptional activity.****

**Tetramerization domain (TD)**- Enables p53 to form tetramers, essential for its biological function.

**C-terminal regulatory domain (CTD) **- Involved in modulating DNA-binding activity and interactions with other proteins.

** **

**_P53 function_**__

P53 is a tetramer that binds to DNA and activates the expression of genes involved in cell cycle arrest, DNA repair, senescence and apoptosis. Its ability to sense and respond to DNA damage makes it a pivotal player in maintaining genomic stability. However, the protein is often mutated in cancer, with the majority of mutations located in its DNA-binding domain, disrupting its function and leading to unchecked cell proliferation.

 

**_Evaluation of Homology Modeling and AlphaFold on p53 Protein_.**

Homology modeling is performed using templates from known structures of homologous proteins. Homology modeling relies on the availability of closely related template structures and works well for conserved regions, but it struggles with disordered or unique regions like those in p53. In contrast, AlphaFold3 can predict protein structures _de novo_, offering greater accuracy across both structured and disordered regions, even without a template. While homology modeling is effective when high-quality templates are available, AlphaFold3 is more versatile and generally produces more reliable models. For proteins with complex or flexible regions, such as p53, AlphaFold3 outperforms homology modeling in accuracy and applicability. AlphaFold3, due to its reliance on advanced machine learning techniques, predicts the entire structure of the p53 protein, including regions for which no structural data is available. It makes use of both evolutionary information and physical constraints to produce highly accurate models.

<!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Picture_x0020_1" o:spid="_x0000_i1027" type="#_x0000_t75"
 style='width:462pt;height:191pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="file:///C:/Users/pranjal/AppData/Local/Packages/oice_16_974fa576_32c1d314_1138/AC/Temp/msohtmlclip1/01/clip_image001.jpg"
  o:title="" croptop="5089f" cropbottom="10542f"/>
</v:shape><![endif]--><!--[if !vml]-->![](file:///C:/Users/pranjal/AppData/Local/Packages/oice_16_974fa576_32c1d314_1138/AC/Temp/msohtmlclip1/01/clip_image002.jpg)<!--[endif]-->

**Figure 1. Predicted structures of p53 generated by (A) Swiss Model and (B) Alpha Fold3.**

** **

**_Protein alignment with PyMol_**

PyMOl is an open-source visualization software that analyses 3D structures of macromolecules. We used PyMol to superimpose multiple structure of P53 so that we could identify the most likely conformation of the P53 modelled by AlphaFold and Swiss Model. One of the major shortcomings of AlphaFold3 modelling is its variable performance in predicting proteins which undergo significant conformational changes depending on ligand attachment (apo, agonist and antagonist). By performing RMSD analysis in PyMOl we can quantify how closely a modelled structure aligns with an experimentally determined structure.

 

 

 

 

 

 

**Table 1. RMSD scores generated from the alignment of modelled p53 structures and p53 reference crystal structures from RCSB PDB.**

|                                                 |                                                      |                                 |                 |                          |
| ----------------------------------------------- | ---------------------------------------------------- | ------------------------------- | --------------- | ------------------------ |
| **PDB IDs of P53 reference crystal structures** | **Conformation of P53 reference crystal structures** | **Aligned Structures in PyMol** | **RMSD Scores** | **No. of Atoms matched** |
| 8UQS                                            | Apo                                                  | 8QUS vs Swiss Model             | 0.903           | 115                      |
|                                                 |                                                      | 8QUS vs AlphaFold3              | 18.670          | 111                      |
| 3TS8                                            | Agonist                                              | 3TS8 vs Swiss Model             | 28.805          | 127                      |
|                                                 |                                                      | 3TS8 vs AlphaFold3              | 0.515           | 192                      |
| 6MXY                                            | Antagonist                                           | 6MXY vs Swiss Model             | 17.432          | 106                      |
|                                                 |                                                      | 6MXY vs AlphaFold3              | 8.955           | 38                       |

** **

Three P53 crystal structures (PDB IDs: 8UQS, 3TS8 and 6MXY) with the respective apo, agonist and antagonist protein conformations were aligned against Swiss Model and AlphaFold3 protein models. The root-mean-square deviation (RMSD) scores generated from the alignments in PyMol are presented in Table 1. RMSD scores in PyMol are calculated by superimposing two 3D models of macromolecules or proteins and a reference structure to provide insight to the accuracy of the predicted models, structural similarity and protein conformational changes. It measures the average distance between the atoms (typically the backbone or Cα atoms) of the two aligned structures. Low RMSD values > 2 Å signify high structural similarity suggesting that the modelled protein approximates well the experimentally obtained reference structure (from NMR or X-ray crystallography). 

The alignment of 8UQS and the p53 Swiss model structures yielded an RMSD score of 0.903 Å across 115 atoms. An RMSD value below 1 Å indicates that strong similarity between the structures for the apo conformation of p53. Compared to the crystal structures 3TS8 and 6MXY the Swiss model p53 predicted structure yielded 28.805 Å and 17.432 Å RMSD scores, respectively (refer to Table 1). These results imply that the Swiss model most likely predicted a protein with a conformation closely resembling the apo structure of P53 from the reference 8UQS.

**<!--[if gte vml 1]><v:shape id="Picture_x0020_3"
 o:spid="_x0000_i1026" type="#_x0000_t75" style='width:468pt;height:249.5pt;
 visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="file:///C:/Users/pranjal/AppData/Local/Packages/oice_16_974fa576_32c1d314_1138/AC/Temp/msohtmlclip1/01/clip_image003.jpg"
  o:title=""/>
</v:shape><![endif]--><!--[if !vml]-->![](file:///C:/Users/pranjal/AppData/Local/Packages/oice_16_974fa576_32c1d314_1138/AC/Temp/msohtmlclip1/01/clip_image004.jpg)<!--[endif]-->**

**Figure 2. RMSD analysis of P53 crystal structure (PDB ID: 8UQS) aligned with Swiss Model P53 structure.**

** **

Meanwhile the 3TS8 crystal structure aligned with the AlphaFold3 P53 structure yielded an RMSD score of 0.515 Å across 192 atoms. Compared to the crystal structures 8UQS and 6MXY the AlphaFold3 P53 model yielded 18.670 Å and 8.955 Å RMSD scores, respectively (see Table 1). This low score suggests that AlphaFold3 accurately predicted the conformation of the P53 agonist bound structure (PDB ID: 3TS8). Both homology modelling and AlphaFold3 struggle to provide accurate predictions for the antagonist bound structure (PDB ID: 6MXY).

<!--[if gte vml 1]><v:shape
 id="Picture_x0020_2" o:spid="_x0000_i1025" type="#_x0000_t75" style='width:468pt;
 height:235.5pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="file:///C:/Users/pranjal/AppData/Local/Packages/oice_16_974fa576_32c1d314_1138/AC/Temp/msohtmlclip1/01/clip_image005.jpg"
  o:title=""/>
</v:shape><![endif]--><!--[if !vml]-->![](file:///C:/Users/pranjal/AppData/Local/Packages/oice_16_974fa576_32c1d314_1138/AC/Temp/msohtmlclip1/01/clip_image006.jpg)<!--[endif]-->

**Figure 3. RMSD analysis of P53 crystal structure (PDB ID: 3TS8) aligned with AlphaFold3 P53 model**

In conclusion, the RMSD analysis shows that the accuracy of predicting the conformation of P53 varies with the protein binding state. For this study, AlphaFold outperformed Swiss Model in capturing the agonist bound structure of P53, while Swiss model was a better predictor of the apo conformation. Overall, AlphaFold3 gave more reliable results with lower RMSD scores across all three conformations of the reference structures.

 

**_Hyperlinks of p53 models:_**

[**https://swissmodel.expasy.org/interactive/NZmDQR/models/**](https://swissmodel.expasy.org/interactive/NZmDQR/models/)**__**

[**https://alphafold.com/entry/G3WS63**](https://alphafold.com/entry/G3WS63)**__**

**_ _**

**_References_**__

 Levine, A. J. (1997). p53, the Cellular Gatekeeper for Growth and Division. _Cell_, 88(3), 323-331. <https://doi.org/10.1016/S0092-8674(00)81871-1>.

Zhang, Y. (2008). Progress and Challenges in Protein Structure Prediction. _Current Opinion in Structural Biology_, 18(3), 342-348. <https://doi.org/10.1016/j.sbi.2008.02.008>.

 Cho, Y., Gorina, S., Jeffrey, P. D., & Pavletich, N. P. (1994). Crystal Structure of a p53 Tumor Suppressor-DNA Complex: Understanding Tumorigenic Mutations. _Science_.   <https://doi.org/10.1126/science.8023157>.

Schrödinger, LLC. (2015). The PyMOL Molecular Graphics System, Version 1.8. (<https://pymol.org/2/>)

Jumper, J., Evans, R., Pritzel, A., et al. (2021). Highly Accurate Protein Structure Prediction with AlphaFold. _Nature_. <https://doi.org/10.1038/s41586-021-03819-2>.

Joerger, A. C., & Fersht, A. R. (2008). Structural Biology of the Tumor Suppressor p53. _Annual Review of_ _Biochemistry_, 77(1), 557-582. <https://doi.org/10.1146/annurev.biochem.77.060806.091238>.

 

 

 

 

 
